src/frequenz/sdk/actor/_power_managing/_matryoshka.py

```diff
@@ -36,6 +36,60 @@
 _logger = logging.getLogger(__name__)
 
 
+# Just 20 lines of code in this function, but unfortunately 8 of those are return
+# statements, and that's too many for pylint.
+def _clamp_to_bounds(  # pylint: disable=too-many-return-statements
```

Comment:
I wonder if we should remove this check completely, I don't see anything wrong in having many `returns`, I think is good for readability to avoid deep nesting and it helps short-circuiting by discarding simple cases first.

Reply:
@llucax is this for repo config?

Reply:
Yeah, if we agree about it.